lib/pbench/server/auth/auth.py

```diff
@@ -120,7 +177,10 @@ def verify_auth(auth_token):
                     e,
                 )
         except jwt.InvalidTokenError:
-            pass  # Ignore this silently; client is unauthenticated
+            Auth.logger.warning(
```

Comment:
This should probably be `debug`, or at most `info` (and only because the condition will be going away when we drop the internal users); this isn't an "unusual" condition.